src/components/NewsFeed.js

```diff
@@ -93,18 +129,53 @@ const NewsList = ({ news }) => {
 
 const NewsItem = ({ newsItem }) => {
```

Comment:
Shouldn't this be it's own component file?

Reply:
That depends. If you look at it it has 6 lines of code. Do we want to jump through 3-4 components in different files instead of having them in one place? From my perspective those are already prepared to be a separate components (same as NewsItem for example) we can put them in separate files whenever we want but since we are still in development phase of this feature it is a bit easier to navigate when they are in one file..
Will we remember to put them in separate files at the end? Probably not... But also if some of it will grow too much I will add as separate component.

Reply:
I think for me it more has to do with modularity where the code is easy to reuse for another purpose later. But this is maybe fine for now..